testsuite/tck/pom.xml

```diff
@@ -101,6 +101,7 @@
           <skip>${skipTck}</skip>
           <suiteXmlFiles>
             <suiteXmlFile>${project.build.directory}/test-classes/tck-base-suite.xml</suiteXmlFile>
+            <suiteXmlFile>${project.build.directory}/test-classes/smallrye-suite.xml</suiteXmlFile>
```

Comment:
We don't need it

Reply:
Well, I don't feel comfortable in doing changes that should be retro compatible without any tests :)

Reply:
OK, lets just move it into testsuite/basic

Reply:
Ok :)

Reply:
Ok, I've had another idea and removed this. I've just implemented a test to make sure that the values match if you use different config properties. The behaviour will be tested by the TCK.